### ISSUE TITLE
Customing PR header not functional

### ISSUE THREAD

**Issue by pilotak** [google-github-actions/release-please-action#663]
### TL;DR

I wanted to chage the **beep** **boop** so i found in docu https://github.com/googleapis/release-please/blob/main/docs/customizing.md that it can be done `pull-request-header` so i did, but this option is not supported 

### Action YAML

```yaml
name: release

on:
  push:
    branches:
      - master

jobs:
  release_helper:
    runs-on: ubuntu-latest
    outputs:
      release: ${{ steps.release.outputs.release_created }}
      version: ${{ steps.release.outputs.tag_name }}
    steps:
      - name: Checkout
        uses: actions/checkout@v3

      - uses: google-github-actions/release-please-action@v3
        id: release
        with:
          release-type: node
          pull-request-header: ":robot: New release version"
          pull-request-title-pattern: "chore: release${component} ${version}"
```

### Log output

```text
Unexpected input(s) 'pull-request-header', valid inputs are ['token', 'fork', 'clean', 'package-name', 'release-type', 'bump-minor-pre-major', 'bump-patch-for-minor-pre-major', 'path', 'changelog-path', 'changelog-host', 'command', 'version-file', 'extra-files', 'default-branch', 'changelog-types', 'config-file', 'manifest-file', 'signoff', 'github-api-url', 'github-graphql-url', 'repo-url', 'monorepo-tags', 'pull-request-title-pattern', 'draft', 'draft-pull-request', 'changelog-notes-type', 'versioning-strategy', 'release-as', 'skip-github-release', 'prerelease', 'component', 'include-v-in-tag', 'tag-separator', 'snapshot-labels', 'bootstrap-sha', 'last-release-sha', 'always-link-local', 'separate-pull-requests', 'plugins', 'labels', 'release-labels', 'skip-labeling', 'sequential-calls', 'group-pull-request-title-pattern', 'release-search-depth', 'commit-search-depth', 'proxy-server']
```

**Comment by nhuray** [comment]
@pilotak It looks like the action does not have that input at the moment but a PR is opened [here](https://github.com/google-github-actions/release-please-action/pull/615)

**Comment by pilotak** [comment]
ok, that's a pitty it has not been merged yet

**Comment by chingor13** [comment]
@nhuray This feature should have worked via the manifest config. Can you link to the action config?

I highly recommend switching to the manifest config as we are continually adding new features/configuration options and it's not ideal to try and allow every configuration option via GitHub actions inputs.

**Comment by nhuray** [comment]
@chingor13 I agree with you, it's better to use the manifest config.

But I did that config on a private repo and some configuration are not considered: 

`.github/workflows/release-please.yaml`
```
name: release-please
jobs:
    release-please:
        runs-on: ubuntu-latest
        steps:
            - uses: google-github-actions/release-please-action@v3
              id: release
              with:
                  release-type: node
                  token: ${{ secrets.GITHUB_TOKEN }}
                  command: manifest
                  monorepo-tags: true
                  config-file: .release/config.json
                  manifest-file: .release/manifest.json
```

`.release/config.json`
```
{
    "bootstrap-sha": "bbe88cfe169cf45a59e4a9cb8efa082caea5f140",
    "pull-request-header": ":rocket: New Release",
    "include-v-in-tag": false,
    "label": "status: draft",
    "release-label": "status: released",    
    "packages": {
        "apps/a": {
            "package-name": "a"
        },
        "apps/b": {
            "package-name": "ab
        }
    }
}
```

In my tests, I could not find a way to have that config working:
 - `include-v-in-tag: false`: The releases are created with the `v` prefix 
 - `pull-request-header`: The default header is not replaced by the one specified

I also noticed some discrepancies with [singular](https://github.com/googleapis/release-please/blob/main/schemas/config.json#L349) and [plural](https://github.com/google-github-actions/release-please-action/blob/main/action.yml#L142) for the labels.

I'm going to dig more in detail in the code tomorrow to understand why my configuration is not taking in consideration.

**Comment by pilotak** [comment]
@chingor13 I can understand it's better to use a manifest, but there is an open PR for this. Can it be merged?

**Comment by chingor13** [comment]
released in 3.7.0
